examples/multiple_on_off_plugin_units/main/app_driver.cpp

```diff
@@ -85,4 +89,53 @@ esp_err_t app_driver_attribute_update(app_driver_handle_t driver_handle, uint16_
     return err;
 }
 
+app_driver_handle_t app_driver_button_init()
+{
+#ifdef CONFIG_USER_BUTTON
+    s_reset_gpio = (gpio_num_t)CONFIG_USER_BUTTON_GPIO;
```

Comment:
```suggestion
    s_reset_gpio = (gpio_num_t)CONFIG_USER_BUTTON_GPIO;
```

`s_` says that the variable is static but it is extern. I'd suggest we add an out arg to this API and get the reset gpio pin number. Or add another API which determines the reset gpio pin?